proprietary/Components/src/denhaag/link-grid.tokens.json

```diff
@@ -0,0 +1,45 @@
+{
+  "denhaag": {
+    "link-grid": {
+      "panel": {
+        "background-color": { "value": "{denhaag.color.ocher.3}" },
+        "background-image-width": { "value": "41.66%" },
+        "padding": {
+          "xs": { "value": "48px" },
```

Comment:
Omdat we met de tokens werken, moeten deze volgensmij ook uit andere tokens komen.

Er zijn denhaag spacing tokens (losse JS file). die je kan gebruiken

Reply:
proprietary/Common/src/denhaag/space/space.tokens.json

Reply:
de "panel" kun je weglaten en direct in de --denhaag-link-grid stoppen, omdat (naar mijn weten) geen .denhaag-link-grid-panel class in de html zit.

Reply:
Done in de ge-update PR!